cluster-api/gcp-deployer/cmd/create.go

```diff
@@ -63,13 +69,14 @@ func RunCreate(co *CreateOptions) error {
 		return err
 	}
 
-	d := deploy.NewDeployer(provider, kubeConfig)
+	d := deploy.NewDeployer(provider, kubeConfig, co.MachineSetup)
 
 	return d.CreateCluster(cluster, machines)
 }
 func init() {
 	createCmd.Flags().StringVarP(&co.Cluster, "cluster", "c", "", "cluster yaml file")
 	createCmd.Flags().StringVarP(&co.Machine, "machines", "m", "", "machine yaml file")
+	createCmd.Flags().StringVarP(&co.MachineSetup, "setup", "s", "", "machine setup configs yaml file")
```

Comment:
(nit)
I would keep it as machinesetup or machinessetupconfig. setup by itself is a bit confusing.